### ISSUE TITLE
`C_Cpp.errorSquiggles` disabled by default

### ISSUE THREAD

**Issue by RedCMD** [yannis300307/CasioDevTools#18]
I see you disable `C_Cpp.errorSquiggles` by default
was that because `C_WHITE` etc shows up as errors?
![image](https://github.com/yannis300307/CasioDevTools/assets/33529441/9a043051-67e3-4cb9-b082-296e9a848930)

it is because gint only defines things when a model is selected like: `#define FXCG50`

![image](https://github.com/yannis300307/CasioDevTools/assets/33529441/06bd9fa0-cc73-464f-a641-b628e442e7ae)

**Comment by yannis300307** [comment]
Yes it's a problem with the definitions but I don't know if it's a good idea to define FXCG50 at the top of each programs... I have to find a way to fix this issue dirrectly with the VS code config.
